Pratiksha/html/PratikshaA4_2.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        body{
+            background-color: cadetblue;
+        }
+        p,h1{
+            text-align: center;
+        }
+        table,td,th{
+           width: 70%;
+
+            align-content: center;
+            
+            border : 2px solid green;
+            border-collapse: collapse;
+            padding: 10px;
+            margin: 10px
+    
+        }
+        th{
+            color: blue;
+        }
+    </style>
+</head>
+<body>
+    <h1>Assignment 4</h1>
+    
+    <p>A test cell with merged cells</p>
+    
+    <table>
+        <tr>
+            <th colspan="2" rowspan="2">
```

Comment:
Here write on rowspan only 
Not colspan ok

Reply:
this line is correct MKaddeer ji